docs/clients/snappi.md

```diff
@@ -0,0 +1,711 @@
+# snappi
+
+![snappi](https://github.com/open-traffic-generator/snappi/raw/main/snappi-logo.png)
+
+## Overview
+
+Test scripts written in `snappi`, an auto-generated Python module, can be executed against any traffic generator conforming to the Open Traffic Generator API.
+
+## Install on a client
+
+```sh
+python -m pip install --upgrade snappi
+```
+
+## JSON
+
+Every object in snappi can be serialized to or deserialized from a JSON string which conforms to [Open Traffic Generator API](https://github.com/open-traffic-generator/models). This facilitates storing traffic configurations as JSON files and reusing them in API calls with or without further modifications.
+
+* Create a sample config
+
+```python
+import snappi
+api = snappi.api()
```

Comment:
Should show to initialize for a gRPC endpoint
> api = snappi.api(location="localhost:40051", transport=snappi.Transport.GRPC)

Reply:
We have that in the `hello.py` example at the bottom:
import snappi
```
# create a new API instance where location points to controller.
# this will use HTTP transport by default; in order to use gRPC instead,
# one can pass additional kwarg `transport=snappi.Transport.GRPC`
api = snappi.api(location='https://localhost')
```

Reply:
added to this snippet as well